tests/go.mod

```diff
@@ -4,11 +4,11 @@ go 1.19
 
 require (
 	github.com/rancher/hull v0.0.0-20230424152137-627ef5347afd
-	github.com/rancher/wrangler v1.0.0
+	github.com/rancher/wrangler v1.0.1
 	github.com/stretchr/testify v1.8.1
 	k8s.io/api v0.26.0
 	k8s.io/apimachinery v0.26.0
-	k8s.io/kubernetes v1.26.3
+	k8s.io/kubernetes v1.15.0-alpha.0
```

Comment:
@nicholasSUSE A downgrade?

Reply:
Just tested everything locally, and it is all still working. 
This downgrade seems to be due to a dependabot that evaluates security vulnerabilities. 
Still I am getting in contact with the Sec team to understand this better.